src/platform/backends/libvirt/libvirt_virtual_machine.cpp

```diff
@@ -278,7 +278,26 @@ void mp::LibVirtVirtualMachine::start()
     update_state();
 
     if (virDomainCreate(domain.get()) == -1)
-        throw std::runtime_error(virGetLastErrorMessage());
+    {
+        state = State::suspended;
+        update_state();
+
+        std::string error_string{virGetLastErrorMessage()};
+        if (error_string.find("virtio-net-pci.rom: 0x80000 in != 0x40000") != std::string::npos)
```

Comment:
Just a question: are you sure these will always be the exact numbers in the error?

Reply:
Yes, the ROM size doubled between the `core16` and `core18` libvirt versions, and we only want to catch this exact case here. If we get any other message, we want to surface it as-is.